src/sfctl/tests/chaos_schedule_test.py

```diff
@@ -0,0 +1,555 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom chaos schedule command related tests"""
+import unittest
+import sfctl.custom_chaos_schedule as sf_c
+
+#pylint: disable=invalid-name,line-too-long
+
+class ChaosScheduleTests(unittest.TestCase):
```

Comment:
Thanks for adding all the tests!! Can you also add an entry to "request_generation_test"? Thanks!